CIP-ledger-evolution/Serialization.md

```diff
@@ -0,0 +1,97 @@
+# Serialization in the ledger
+
+This document specifies a policy for the backwards compatibility of the serialization scheme of
```

Comment:
This document doesn't obviously fit into the flow of the main document. I'm also unsure that you need it. I think all you really need to say in the main document is something like:
- Changes to the transaction format usually require serialization changes
- We can also change the serialization for other reasons
- Changing the serialization requires a new ledger era (or whatever)

Reply:
yea, sorry, I was being lazy. This is CIP-80, where the discussion for a legitimizing ledger category CIP first came up. maybe I'll remove it and mention CIP-80.